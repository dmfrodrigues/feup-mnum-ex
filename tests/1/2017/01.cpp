#include <bits/stdc++.h>
using namespace std;
double g(double x){
    double x_ = x+1.2;
    double c = cos(x_);
    double s = sin(x_);
    double d = c*c;
    double up = -d*(3.0L*x*s+c)+3.6L;
    double dw = 1.0L-3.0L*d*s;
    return up/dw;
}
int main(){
    double x; cin >> x;
    cout << setprecision(10) << fixed << g(x) << endl;
    return 0;
}
