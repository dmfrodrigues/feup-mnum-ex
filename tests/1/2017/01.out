3.7633057619
